content/fis-ii/1000-formulario.md

```diff
@@ -315,24 +323,86 @@ $K = 1 + \chi$
 
 $k$ - Constante Dielétrica do Meio - Capacitância aumenta por fator $k$
 
-## Magnetostática
+## Corrente
 
 $
-\vec j = Nq \vec v
+\vec J = Nq \vec v
 $
 
-$\vec j$ - Densidade de Corrente $\ \ N$ - Número de Cargas $\ \ \vec v$ - Velocidade (média)
+$\vec J$ - Densidade de Corrente $\ \ N$ - Número de Cargas $\ \ \vec v$ - Velocidade (média)
 
 $
-I = \int_{S} \vec j \ d \vec S
+I = \int_{S} \vec J \ d \vec S
 $
 
-$I$ - Corrente Elétrica $\ \ \vec j$ - Densidade de Corrente $\ \ S$ - Superfície
+$I$ - Corrente Elétrica $\ \ \vec J$ - Densidade de Corrente $\ \ S$ - Superfície
 
 $
-\vec j = \sigma \vec f
+\vec J = \sigma \vec f
 $
 
-$\vec j$ - Densidade de Corrente $\sigma$ - Condutividade do Meio (Constante) $\vec f$ - força
+$\vec J$ - Densidade de Corrente $\ \ \sigma$ - Condutividade do Meio (Constante) $\vec f$ - força
 
 ### Lei de Ohm
+
+$R = \cfrac{V}I$
+
+$R$ - Resistência Elétrica $\ \ V$ - Difernça de Potencial $\ \ I$ - Corrente Elétrica
+
+### Lei de Joule
+
+$P = VI = RI^2$
+
+$P$ - Potência Emitida (Dissipada) $\ \ V$ - Difernça de Potencial\
+$I$ - Corrente Elétrica $\ \ R$ - Resistência Elétrica
+
+## Magnetostática
+
+$\vec F = Q(\vec E + \vec v \times \vec B)$
+
+$\vec F$ - Força de Lorentz $\ \ Q$ - Carga $\ \ \vec E$ - Campo Elétrico\
+$\vec v$ - Velocidade da Carga $\ \ \vec B$ - Campo Magnético
+
+### Lei de Biot-Savart
+
+$
+\vec B (\vec r) = \cfrac {\mu_{0}}{4 \pi} \int_{C} \cfrac{\vec I \times \vec e_{r}}{r^2} \ dl  '
+$
+
+$vec B (\vec r)$ - Campo Magnético $\ \ \vec I$ - Corrente Elétrica $\ \ r$ - Distância a um Ponto
+
+### Lei de Ampére
```

Comment:
```suggestion
### Lei de Ampère
```

Reply:
https://en.wikipedia.org/wiki/Andr%C3%A9-Marie_Amp%C3%A8re
No it isn't